runtime/src/bank.rs

```diff
@@ -6265,7 +6264,23 @@ impl Bank {
         self.rewards_pool_pubkeys =
             Arc::new(genesis_config.rewards_pools.keys().cloned().collect());
 
+        self.apply_feature_activations(
```

Comment:
Was it needed to be moved before the next code block?

Reply:
Yes, otherwise create_program_runtime_environment() does not see the correct feature set.